docs/services/esi/index.md

```diff
@@ -5,34 +5,49 @@ title: ESI
 
 ## User-Agent
 
-All ESI Requests should be accompanied by an appropriate User-Agent request header providing a method for CCP to contact the source of the request in the event of an issue.
+All ESI Requests **should** be accompanied by an appropriate User-Agent information providing a method for CCP to contact the source of the request in the event of an issue.
 
-This could be one or more of the following
+### Data transmission
+
+That data should be transmitted in, by order of preference 
+ 1. A `User-Agent` header in the request
+ 2. A `X-User-Agent` header instead, if that previous one can't be set reliably.
+ 3. A `user_agent` query parameter, if you can't set your headers reliably.
```

Comment:
Would be more consistent to keep the same voice here, either "if you can't sent the previous one reliably" in point 2, or "if headers can't be set reliably" in point 3.

Reply:
yeah it's spec, should be neutral.